pkg/attestation/filter.go

```diff
@@ -0,0 +1,51 @@
+package attestation
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/antonmedv/expr"
+)
+
+// FilteredEnv type including an array of Attestations
+type FilteredEnv struct {
```

Comment:
```suggestion
type AttestationFilterEnv struct {
```

Reply:
I like the suggestion (this is how it was before), but Linter is not happy when I change the naming to AttestationFilterEnv, that's why I changed it in https://github.com/philips-labs/fatt/pull/8/commits/43604f5f45117a477c420928ff49de0f299d33f6